pennylane/workflow/qnode.py

```diff
@@ -590,7 +589,16 @@ def __copy__(self):
 
     def __repr__(self):
         """String representation."""
-        return f"<QNode: device='{self.device}', interface='{self.interface}', diff_method='{self.diff_method}'>"
+        if not isinstance(self.device, qml.devices.LegacyDeviceFacade):

```

Comment:
Why are negated conditions being used? I think it's a bit more readable to change the predicate to `if isinstance(self.device, qml.devices.LegacyDeviceFacade)`.